### ISSUE TITLE
bug report of 1.2.4

### ISSUE THREAD

**Issue by neove** [webpack-contrib/cache-loader#46]
![3d7b6a589c2da27f9dd2b87a3aa02c98](https://user-images.githubusercontent.com/18530075/47773816-46bbab00-dd26-11e8-8f2c-bcbc9f3cfa69.jpg)

**Comment by neove** [comment]
when i update cache-loader from 1.2.2 to 1.2.4， i have this error

**Comment by valera33** [comment]
+1, same here

**Comment by mihiic** [comment]
Doesn't work as part of the angular build as well:
```
ERROR in ./node_modules/@angular/core/fesm5/core.js
Module build failed: TypeError: Cannot read property '_statSync' of undefined

 at stat (/home/m/xbet/mobile/node_modules/@ngtools/webpack/src/virtual_file_system_decorator.js:26:29)
    at toDepDetails (/home/m/xbet/mobile/node_modules/cache-loader/dist/index.js:66:5)
    at arrayIterator (/home/m/xbet/mobile/node_modules/neo-async/async.js:3780:9)
    at timesSync (/home/m/xbet/mobile/node_modules/neo-async/async.js:2292:7)
    at Object.mapLimit (/home/m/xbet/mobile/node_modules/neo-async/async.js:3775:5)
    at Array.<anonymous> (/home/m/xbet/mobile/node_modules/cache-loader/dist/index.js:89:18)
    at arrayEachFunc (/home/m/xbet/mobile/node_modules/neo-async/async.js:2512:19)
    at Object.parallel (/home/m/xbet/mobile/node_modules/neo-async/async.js:6867:9)
    at Object.loader (/home/m/xbet/mobile/node_modules/cache-loader/dist/index.js:88:9)
```

Explicitly stating version in package.json to 1.2.2 fixes the problem.

**Comment by valera33** [comment]
@mihiic yep, I did the same

**Comment by zzzJH** [comment]
same here

**Comment by whxaxes** [comment]
same here !!!

**Comment by michael-ciniawsky** [comment]
Please try

**`node_modules/cache-loader/index.js`**
```diff
// L69
- }
+ }.bind(this);
```

**Comment by whxaxes** [comment]
@michael-ciniawsky  see https://github.com/webpack-contrib/cache-loader/commit/b84d13e#r31116413

**Comment by patriceo** [comment]
Hello, having the same here, breaking my angular project build :

`ERROR in ./node_modules/core-js/es6/symbol.js
Module build failed: TypeError: Cannot read property '_statSync' of undefined
    at stat (/home/circleci/wfo/wfo-web-client/node_modules/@ngtools/webpack/src/virtual_file_system_decorator.js:26:29)
    at toDepDetails (/home/circleci/wfo/wfo-web-client/node_modules/cache-loader/dist/index.js:66:5)
    at arrayIterator (/home/circleci/wfo/wfo-web-client/node_modules/neo-async/async.js:3780:9)
    at timesSync (/home/circleci/wfo/wfo-web-client/node_modules/neo-async/async.js:2292:7)
    at Object.mapLimit (/home/circleci/wfo/wfo-web-client/node_modules/neo-async/async.js:3775:5)
    at Array.<anonymous> (/home/circleci/wfo/wfo-web-client/node_modules/cache-loader/dist/index.js:89:18)
    at arrayEachFunc (/home/circleci/wfo/wfo-web-client/node_modules/neo-async/async.js:2512:19)
    at Object.parallel (/home/circleci/wfo/wfo-web-client/node_modules/neo-async/async.js:6867:9)
    at Object.loader (/home/circleci/wfo/wfo-web-client/node_modules/cache-loader/dist/index.js:88:9)`

**Comment by kartheininger** [comment]
@patriceo :
If you need to fix it fast, add `"cache-loader": "1.2.2",` to your `package.json`
If you have time, let the guys here release a new version (1.2.4 or 1.2.5 i guess). This will get fetched automatically, because in webpack there is a dependency to `"cache-loader": "^1.2.0"`

**Comment by patriceo** [comment]
> @patriceo :
> If you need to fix it fast, add `"cache-loader": "1.2.2",` to your `package.json`
> If you have time, let the guys here release a new version (1.2.4 or 1.2.5 i guess). This will get fetched automatically, because in webpack there is a dependency to `"cache-loader": "^1.2.0"`

Thanks that's exactly what I did, looking forward to seeing a new version.

**Comment by michael-ciniawsky** [comment]
Fixed by #47 

Released in [`v1.2.5`](https://github.com/webpack-contrib/cache-loader/releases/tag/v1.2.5)
